optuna_dashboard/_app.py

```diff
@@ -306,6 +307,28 @@ def post_preference(study_id: int) -> dict[str, Any]:
         response.status = 204
         return {}
 
+    @app.post("/api/studies/<study_id:int>/component")
+    @json_api_view
+    def post_component(study_id: int) -> dict[str, Any]:
```

Comment:
Could you update the view function's name and api endpoint to elaborate the role of this endpoint?